style.css

```diff
@@ -68,7 +68,55 @@
   cursor:not-allowed;
 }
 
+.fit-content {
+  max-width: fit-content;
+}
+
+.embedded-safe {
```

Comment:
1. Továbbra is úgy érzem, hogy .embedded-safe és .embedded classok között jelentős átfedés van, megoldásnak pl. közös class használata
2. Azt hiszem azt beszéltük, hogy ne legyen border.